multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -205,6 +221,20 @@ def parse_cov_dist(self):
                         if cumcov > 1:  # require >1% to prevent long flat tail
                             xmax = max(xmax, x)
                     else:  # for per-contig plot
+                        # filter out contigs based on exclusion patterns
+                        if any(fnmatch.fnmatch(contig, pattern) for pattern in exclude_contigs):
+                            # Commented out since this could be many thousands of contigs fo reach!
+                            # log.debug("Skipping excluded contig '{}'".format(contig))
+                            continue
```

Comment:
What I have done for similar cases in the past is make these debug statements disabled by default, but have a config option to enable them. Can very occasionally be useful for debugging and not tricky to add.

```suggestion
                            if config.mosdepth_config.get('log_contig_exclusion'):
                                log.debug("Skipping excluded contig '{}'".format(contig))
```